packages/browser-sdk/src/lib/react/useRoomConnection/types.ts

```diff
@@ -24,16 +25,6 @@ export type ScreenshareState = Screenshare;
 type LocalScreenshareStatus = "starting" | "active";
 
 export type ChatMessage = Pick<SignalChatMessage, "senderId" | "timestamp" | "text">;
-export type ConnectionStatus =
```

Comment:
This might actually require a major version bump. This looks like api-breaking change. Whoever imported the `ConnectionStatus` from there, their code won't compile now.

Reply:
Talked to Kevin, he will double-check that.

Reply:
Ok we are good, this is exported from the file but not module so its not part of the api for `browser-sdk`